robottelo.properties.sample

```diff
@@ -53,12 +53,14 @@ provisioning_server=
 image_dir=/opt/robottelo/images
 
 [docker]
-# Internal docker URL in the format http[s]://<server>:<port>. The
+# External docker URL in the format http[s]://<server>:<port>. The
```

Comment:
@oshtaier the external docker URL is specified here. The method is just a wrapper to get this information and add the ability to change the `server_hostname` variable. I'm in favor of leaving the format only here because if, for any reason, it changes we have just one place to update.